### DISCUSSION TITLE
lolMiner 1.50 public beta

### DISCUSSION THREAD

**Discussion by Lolliedieb** [Lolliedieb/lolMiner-releases#1574] [General]
*Windows beta is in testing ... to be released here soon*

### Changes
- 100% LHR unlock for all LHR affected algorithms on LHR v1 and v2 GPUs. Required drivers: 510.60.02 or 510.68 in Linux, 512.15 in Windows. On other drivers the previous 79% unlock on Ethash will be active. 

### Notes:
- The miner will unlock the cards approximately 30 seconds after start. Please make sure all OC operations are completed before. 
- On dual mining the LHR cards will now behave live full hash rate cards. This means dual mining tuning will start approximately 1 Minute after the DAG build is completed.

<hr /><em>This discussion was created from the release <a href='https://github.com/Lolliedieb/lolMiner-releases/releases/tag/1.50%20beta'>lolMiner 1.50 public beta</a>.</em>

**Comment by nirotub** [comment]
Круто. Жду версию для Windows

**Comment by dawidmosk** [comment]
Jeaaa, finally! No everything back work ETH+ALP+OC+FAN :) Uptime 1 hour and 7 minutes and working great for now. Hash rate this same like on nbminer but here i have ALPH with ETH 👯

**Reply by marcefase4** [reply]
estoy extrayendo eth/ton en Windows con lolminers 1.50, ¡el hash de eth sube y baja todo el tiempo! de 74.8(3070ti) a 71mh o 65mh y luego sube!! es muy inestable, ton-coin está fijo en el hash pero eth no. in nbminer i have 79 mh same oc in lol dual 74/71

**Comment by 3ecurityy** [comment]
hi , i test 3080LHR on windows with driver 512.15
1-ton not started after 15min 
2-nbminer accepted share hugly more ...
i love lolminer , i hope it will solved soon

**Reply by smdbg** [reply]
nbminer accepted share about 6% less than lol or rex - nb sux.

**Reply by 3ecurityy** [reply]
> nbminer accepted share about 6% less than lol or rex - nb sux.

after 24h testing nbminer i can say that 
anyway lol will solved soon

**Comment by Linkestis** [comment]
guys I apologize but I'm a total idiot ... how to give the file admin or root rights? win 11 does not allow me

**Reply by 3ecurityy** [reply]
right click on lolminer.exe > showMoreOption > propertis > compatibility (TAB) > run this program as an admi... (Check)

**Comment by Tux0Topo** [comment]
Hi there! It has any issues with mixing FHR and LHR cards? Thanks a lot!!

**Comment by Surreal2Tech** [comment]
Updated to 1.50 and now none of Nvidia cards will start up, I have to go back to 1.46a to get them start.

AMD Cards do not have OC
NVidia Cards do have OC

OS Version:  5.10.0-hiveos #110
Nvidia Driver Version: 510.68.02
AMD Driver Version: 21.40.1 (5.13.0201)
Cards:
RX 6600 × 4
GTX 1660s × 3
GTX 1660 Ti × 1
RX 6700 × 1
RTX 3080 Ti × 1
RTX 2060 × 1

Any Info is greatly appreciated.

<img width="289" alt="image" src="https://user-images.githubusercontent.com/105397035/167961269-2a40458a-184b-41a2-8cc7-404a13d59f27.png">

Extra params: (forgot to add)

<img width="171" alt="image" src="https://user-images.githubusercontent.com/105397035/167962511-e8647873-69d8-44f6-9097-13e7e9dcb2b3.png">

**Reply by Tux0Topo** [reply]
Hi! First of all, try using 2 different miners, one for nvidia, and the other for AMD, When have problems on any rig, start trying that. Also, just for now, i´m trying trex (eth + alp) on FHR, and lolminer for LHR... Maybe later will test LHR + FHR in lolminer, but nbminer/trex/gminer state that mixing card gives problems...

**Reply by Surreal2Tech** [reply]
Okay so I should move my AMD cards to say NBMiner and set lolminer to run only with NVidia., correct?

All cards work great under lolminer 1.46a with out issue.

**Reply by Surreal2Tech** [reply]
Okay before adding another miner, I just unplugged the 2 AMDs out of one machine, a left the Nvidias active.  same issue NVidias are not starting and it has been about 5 minutes of uptime now .

<img width="194" alt="image" src="https://user-images.githubusercontent.com/105397035/167964170-ab62478c-6d69-4997-ae21-3f47bca10657.png">

**Reply by Surreal2Tech** [reply]
So add another miner and split the Nvidia and AMD cards, Same issue NVidia cards are starting with lolminer 1.50

<img width="294" alt="image" src="https://user-images.githubusercontent.com/105397035/167964998-3f3d28bf-54c1-4ab2-8f0b-bac4bc9766b8.png">

**Reply by Tux0Topo** [reply]
Weird... im having trex for a 3080 fhr, teamred for 4x rx5700xt, lolminer for 3080 lhr + 3080ti lhr and gminer for 1650 + 1650s. It took some time for installing all the miners, but finally started to work...
Also, no lhr commands for lolminer:
![lolminer1 50](https://user-images.githubusercontent.com/20643337/167970262-9fccf462-2a26-42a0-8ad2-60ce3139739d.png)

**Reply by Surreal2Tech** [reply]
I will stop dual mine, and see if that changes anything

**Reply by Surreal2Tech** [reply]
Stopping the dual mine, resolved it and let the Nvidia cards start on version 1.50, funny they worked on 1.46a with out issue.

<img width="352" alt="image" src="https://user-images.githubusercontent.com/105397035/168001436-40150a46-27da-4a8a-aa5d-960b7a106ec0.png">

**Reply by Surreal2Tech** [reply]
I will try putting everything back in mixed mode tomorrow and see if they stay running then as well.

**Reply by Tux0Topo** [reply]
Weird... I was getting 113mhs + 985mhs (eth + ton) on the 3080ti. And not remember how much on the 3080....

**Reply by Surreal2Tech** [reply]
No worries, Like I said the issue was with the Nvidias and not the AMDs.

**Reply by Surreal2Tech** [reply]
So going back to Dual mining prohibits the nvidia cards from starting on my system, goofy I see others dual mining with out issue.

<img width="144" alt="image" src="https://user-images.githubusercontent.com/105397035/168306299-2508462f-c9b6-4fef-bb85-08a08180b60b.png">

**Comment by nikhilchoudhary2264** [comment]
hiii

im using MSI 6600xt Mech 2x OC Edition 8gb
recently updated drivers on 28 Apr 2022 22.5.1 on windows

on dual mining im still getting the same hash rates as earlier
32.7 on eth
680 on ton
![57f4e145-8105-4d09-ad3a-3d10b4aa2fbe](https://user-images.githubusercontent.com/105406971/167990004-35bc85b1-11d4-4816-8caa-ef7c5dbb4736.jpg)

i can see max dag is at 6 gb only

Settings
![fbda50df-f504-4eb3-850a-ed65cb197002](https://user-images.githubusercontent.com/105406971/167989473-fd64b3f6-1b72-4300-afeb-8cdf56fabed7.jpg)
![186b79d3-4c72-486f-9d4b-34630b91116a](https://user-images.githubusercontent.com/105406971/167989479-73d80b33-96b8-463a-b988-4143967d290f.jpg)

update if any settings need to be changed to get higher hash rates

thank you

**Reply by Tux0Topo** [reply]
Sorry... but dont see any issue.
The unlocking update, is only for the nvidia LHR cards. So, you should not see any change with RX6600XT cards...

**Reply by nikhilchoudhary2264** [reply]
ok, lets weight for AMD cards update

**Comment by ketgoodgameth** [comment]
It seems that for the dual mine option, the dual mined coin stays at 0 hashes
For my case, Im using a 3080 LHR: 97 mh/s for ETH and 0 for TON

**Comment by marcefase4** [comment]
hello I'm mining eth/ton on windows with lolminers 1.50 the eth hash goes up and down all the time!! from 74.8(3070ti) to 71mh or 65mh and then go up!! it is very unstable, ton-coin is fixed in the hash but eth is not..

**Comment by sheminasalam** [comment]
In my case, the mining stops after some time.
![image](https://user-images.githubusercontent.com/45196177/168302592-4d758b1e-5f71-4c68-a505-cb53532af046.png)

**Reply by sheminasalam** [reply]
I have tried 1.50 but still after some time, mining is stopped even when the miner is not stopped. The average hasrates just becomes zero like the above fig.

**Comment by Asgarth1977** [comment]
So i just updated the lolminer app in nicehash from 1.48 to the 1.50beta, it runs absolutely fine, however i do not believe it takes in to account the overclocks i use.

Current overclock settings in lolminer in my nicehash app --cclk 1110 --mclk 1370 only giving me 48 to 49MH.s max, whilst both Trex and NBMiner-LHR giving me 62 which is the full capacity for my card on these clocks ?

Card i use: MSI RTX 3070 8gb LHR

How do i solve this issue.

PS: Nicehash app is started in admin mode, so i do believe every miner that is included, auto starts in admin mode.

**Comment by marcefase4** [comment]
Hello, I downloaded the latest 1.5 but it does not start mining ton-coin? then there are two files dual miner ton+eth admin lhr and ton+eth dual use both but in neither does ton start!! does anyone know why there are two files to execute? also the eth hash is still unstable

**Reply by marcefase4** [reply]
the truth is very badly optimized the dual calibrates badly ton and eth lowers the hash out of nowhere sometimes ton does not start you have to restart the miner, I had hopes that with the stable version of 1.50 they would correct some problems but I see that it is not like that

**Comment by dotstal** [comment]
1660ti on hiveos does not start dual, writes an error:
"All devices deselected or failed compatiblity check"
on raveos everything is fine and unlock works when there is a fix, otherwise I can't run the rig along with other maps.
I can't even launch a separate ton
does not mine any card 2060, 2060s, 3060

**Reply by dotstal** [reply]
Everything was decided very simply, in the hive the guys were crooked, apparently they did something when building from the sources, downloading directly from here - everything is ok

**Comment by bernardobraga92** [comment]
When setting clock speeds to gpu and memory it works only on the first few minutes, then the clocks go to default values (without overclock). I'm using 3060 Ti LHR and Ubuntu 20.04 with nvidia 510.47.03 driver

**Comment by p777997** [comment]
having a lot of errors and crashes on LHR cards:
`Unrecoverable memory error by GPU 1.

Reboot for this GPU required.

Please check your (memory) OC `

these are happening at very low OC, so might be that the OC isn't really it.

after such an error card still mines, but with lower speed (19 MH/s for 3060 Ti).

on a whole zoo of my LHR cards - this happens to around 25% of them.

any solutions or fixes possible?

**Comment by warmholeiya** [comment]
Miner in previous versions of 1.50 beta was not connected to the Flexpool pool, but by setting the device numbers in the Miner settings, the problem was solved. After updating to beta version 1.50, this problem was solved, but now, after updating to the final version, 1.50, it will not connect to the Flexpool pool at all.
![Screen Shot 25-02-1401 at 02 14 ب ظ](https://user-images.githubusercontent.com/105580506/168466955-6c491bd7-580a-4cf0-a568-5e83f2837c2c.png)

**Comment by p777997** [comment]
"GPU crashed" and "memory error" on 25 of 135 LHR GPUs overnight.

cards run on low specs with minimal OC.

what's specific - it appears to be still mining on these "crashed/error" cards, but with lower speed. look at screenshots guys:

<img width="1311" alt="s1" src="https://user-images.githubusercontent.com/63288302/168467570-977de429-3568-4973-8b5b-62b57e270c2e.png">

<img width="1311" alt="s2" src="https://user-images.githubusercontent.com/63288302/168467578-67897229-a001-42ef-9ca9-8a33e7cdacc4.png">

**Comment by tugrulserhat** [comment]
Is it not possible to overclock gpus using .bat file on lolminer like on t-rex and nbminer ?

**Comment by fawzi27** [comment]
Why lolminer give 1-2 mh lower than other miners like t-rex or gminer?

**Comment by marcefase4** [comment]
Hello, any news on the dual mining fix? It is impossible for me to calibrate my eth+ton gpu well, it takes me 72mh eth and 1300mh toncoin !! very low if I restart the miner it takes me 70 eth and 1500 toncoin my oc are average !! it's 3070 you

**Comment by Lolliedieb** [comment]
See the most recent update to the miner ;)

Am Mo., 16. Mai 2022 um 18:02 Uhr schrieb marcefase4 <
***@***.***>:

**Reply by marcefase4** [reply]
hola gente descargue 1.51 para win ahora el hash es mucho mas estable pero tengo un problema me ponen dualfactor y en mi rig tengo un 1660 super que me marca mucho mas ton hash que antes casi 1700 antes hacia 800 pero estos 1700 no aparecen en el pool es como si no fuera mio, mi hermano tiene un rig con 3 gpu 1660 y en 1.51 todos marcan hash alto en ton pero ninguno reporta en el pool in auto dual fact figura en 64 lo pongo manual para las 1660 y baja el hash pero en pool mhs figura 0 tuve que volver a usar 1.48 para las gpu 1660 super

**Reply by marcefase4** [reply]
![20220520_095857 1340](https://user-images.githubusercontent.com/105501470/169533191-b5f77333-0657-468b-b4e2-89fd7b750cb9.jpg)

**Reply by marcefase4** [reply]
Hello, I installed the 1.51a, now the 1660 super mines me, but after a while the 3070ti mine drops to half!! but not the mining speed !! but what was reported in the pool, dualfact left it in automatic, the miner only started mining well 1 hour later I downloaded what was reported in the ton-coin pool

**Reply by marcefase4** [reply]
![lol](https://user-images.githubusercontent.com/105501470/169672851-eb1447c6-7cac-4fac-aece-5fa2439d4529.jpg)

**Comment by warmholeiya** [comment]
Not all lolminer versions except 1.48 are connected to Flexpool

**Comment by AndreJoker** [comment]
какой драйвер nvidia лучше использовать для 1.51а версии для не LHR карт? поставил 511.79 заметил снижение хешрейта на 10%

**Reply by dawidmosk** [reply]
te spasiba, вы можете переводить для других на translate.google.com

**Comment by Etherstar** [comment]
since 1.51 Dual Mining ETH+TON started going out of whack, I had to go back to 1.46 just to have the ETH+TON Mining working. Tho the LHR Cards are running less Hash Rates. :(

**Comment by Lolliedieb** [comment]
Ton mining has completely ended since all givers are depleted. Newer
versions of the miner therefore turn if off automatically. You can not mine
it any more.

Etherstar ***@***.***> schrieb am Fr., 1. Juli 2022, 06:31:

**Reply by Etherstar** [reply]
> Ton mining has completely ended since all givers are depleted. Newer versions of the miner therefore turn if off automatically. You can not mine it any more. Etherstar ***@***.***> schrieb am Fr., 1. Juli 2022, 06:31:
> […](#)
> since 1.51 Dual Mining ETH+TON started going out of whack, I had to go back to 1.46 just to have the ETH+TON Mining working. Tho the LHR Cards is running less Hash Rates. :( — Reply to this email directly, view it on GitHub <[#1574 (comment)](https://github.com/Lolliedieb/lolMiner-releases/discussions/1574#discussioncomment-3061052)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AJS63RYB3LN3UWTTWMZDVTDVRZYDTANCNFSM5VVSTANA> . You are receiving this because you authored the thread.Message ID: <Lolliedieb/lolMiner-releases/repo-discussions/1574/comments/3061052@ github.com>

Thank you!
